.github/labeler.yml

```diff
@@ -25,6 +25,12 @@ labels:
 - label: "type/deprecate"
   title: "^deprecate.*"
 
+# If it's removed, or ticked.
 - label: "user-facing-changes"
   negate: true
-  body: 'My PR \*\*DOES NOT\*\* contain user-facing changes'
+  body: 'My PR contains user-facing changes'
+- label: "user-facing-changes"
+  body: '- \[x\] My PR contains user-facing changes'
```

Comment:
I'm inspired when I see @yezizp2012 changed **DOES NOT** to **DOES** in https://github.com/risingwavelabs/risingwave/pull/10269 🤣